qenspy/qens.py

```diff
@@ -140,7 +141,7 @@ def pinball_loss_objective(self, param_vec, y, q, tau, tau_groups):
         """
         K = q.shape[1]
         M = q.shape[2]
-        w = unpack_params(self, param_vec, K, M, tau_groups)
+        w = unpack_params(param_vec, K, M, tau_groups)
```

Comment:
Are we going to need `self` here?
```suggestion
        w = self.unpack_params(param_vec, K, M, tau_groups)
```